software/winrm/src/main/java/org/apache/brooklyn/location/winrm/WinRmMachineLocation.java

```diff
@@ -80,6 +81,9 @@
     public static final ConfigKey<String> OPERATION_TIMEOUT = newConfigKeyWithPrefix(BrooklynConfigKeys.BROOKLYN_WINRM_CONFIG_KEY_PREFIX, WinRmTool.OPERATION_TIMEOUT);
     public static final ConfigKey<Integer> RETRIES_OF_NETWORK_FAILURES = newConfigKeyWithPrefix(BrooklynConfigKeys.BROOKLYN_WINRM_CONFIG_KEY_PREFIX, WinRmTool.RETRIES_OF_NETWORK_FAILURES);
     public static final ConfigKey<Boolean> USE_HTTPS_WINRM = WinRmTool.USE_HTTPS_WINRM;
+    public static final ConfigKey<String> VERIFY_WINDOWS_UP = newStringConfigKey("verifyWindowsUp",
+            "Sometimes restart could happen after a Windows VM is provisioned. This could be because of System Upgrade or other." +
+            " Brooklyn will wait 5 minutes for a restart to happen before returning the machine as accessible.", "5m");
```

Comment:
Should we default to null? Do we expect the "normal" case to be that a reboot will not happen? For example, on AWS and GCE then the VM images are kept up-to-date. We can then document the use of `verifyWindowsUp: 5m` for the clouds that need it?